[view] Account view

vctt94 opened the pull request:
This PR closes #1996 

As @linnutee suggested, I updated the default account to `Primary Account` and the imported one to `Locked for Staking`. 

I am also disabling the imported account details when the user have never bought tickets.

matheusd commented on app/components/views/AccountsPage/AccountRow/AccountDetails.js:
```diff
@@ -117,14 +99,27 @@ const AccountsList = ({
               <CopyToClipboard textToCopy={accountExtendedKey} className="account-actions-pubkey-clipboard" />
             </> :
-            <SlateGrayButton className="account-actions-pubkey-button" onClick={onShowPubKey}><T id="account.reveal" m="Reveal"/></SlateGrayButton>
+            <SlateGrayButton className="account-actions-pubkey-button"><T id="account.Hided" m="Hided"/></SlateGrayButton>
```
"Hidden"

matheusd commented on app/components/views/AccountsPage/AccountRow/Row.js:
```diff
@@ -0,0 +1,61 @@
+import "style/Fonts.less";
+import "style/AccountRow.less";
+
+const isImported = ({ accountName }) => accountName === "imported";
```
It's better to check whether it's the imported account by account number (since that's entirely fixed) rather than name.

vctt94 commented:
I see your point, and I think it is makes sense. So I will return it to imported, and if this becomes an issue, we can open a new issue to discuss it, is that fine? 

Also, I am working in the responsiveness on https://github.com/decred/decrediton/issues/1998

That's why I did not deal with it at this PR.

vctt94 commented:
fixed your reviews comments. 

I switched to compare against the imported account's number. For now, I only added a hard coded number, but I will open a new PR organizing better the constants in a single directory and will fix that as well.

xaur commented:
@matheusd just an idea, if "script" is the only thing that can be "imported", how about `Imported Scripts` as a display name for the `imported` account? Sounds techy, but is more descriptive than just "imported".
